build.sbt

```diff
@@ -101,6 +101,25 @@ lazy val chiselFrontend = (project in file("chiselFrontend")).
   dependsOn(coreMacros)
 
 lazy val chisel = (project in file(".")).
+  enablePlugins(BuildInfoPlugin).
+  settings(
+    // We should really be using name.value, but currently, the package is "Chisel" (uppercase first letter)
+    buildInfoPackage := /* name.value */ "chisel3",
+    buildInfoOptions += BuildInfoOption.BuildTime,
+    buildInfoKeys := Seq[BuildInfoKey](buildInfoPackage, version, scalaVersion, sbtVersion),
+    // Move the managed source directory where git won't complain about it,
```

Comment:
Good points. These changes were due to the misunderstanding that we needed to export the generated BuildInfo with the source. We do want to record it as part of the release, but that can be done separately.